Lombiq.HelpfulLibraries.Cli/Lombiq.HelpfulLibraries.Cli.csproj

```diff
@@ -0,0 +1,34 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <Title>Lombiq Helpful Libraries - Command Line Libraries</Title>
+    <Authors>Lombiq Technologies</Authors>
+    <Copyright>Copyright © 2011, Lombiq Technologies Ltd.</Copyright>
+    <Description>Lombiq Helpful Libraries - Command Line Libraries. This project helps with executing command line calls.</Description>
+    <PackageTags>Lombiq;cli;CliWrap</PackageTags>
+    <PackageIcon>NuGetIcon.png</PackageIcon>
+    <RepositoryUrl>https://github.com/Lombiq/Helpful-Libraries</RepositoryUrl>
+    <PackageProjectUrl>https://github.com/Lombiq/Helpful-Libraries/blob/dev/Lombiq.HelpfulLibraries.Cli/Readme.md</PackageProjectUrl>
```

Comment:
Can't this just be:
```suggestion
    <PackageProjectUrl>https://github.com/Lombiq/Helpful-Libraries/Lombiq.HelpfulLibraries.Cli</PackageProjectUrl>
```
It's not in `dev`, anyways, yet, but will be once we merge.

Reply:
No, you're right. That's my oversight.

Reply:
The `blob/dev/` infix still seems unwanted.

Reply:
What's your vendetta against that poor blob? :P I think it's a perfectly normal file URL. For example `Lombiq.HelpfulLibraries.AspNetCore` has a similar `<PackageProjectUrl>`:

- https://github.com/Lombiq/Helpful-Libraries/blob/dev/Lombiq.HelpfulLibraries.AspNetCore/Readme.md

If you remove it like this, you get a NotFound error:

- https://github.com/Lombiq/Helpful-Libraries/Lombiq.HelpfulLibraries.AspNetCore/Readme.md

Indeed, if you browse to this file from https://github.com/Lombiq/Helpful-Libraries using the GitHub web UI, you get the same blobby URL.

Reply:
I have learned something new - again. Boy, this is getting out of hand!

I actually wasn't aware of this infix having to be there.